docs/content/en/docs/Reference/image_matrix.md

```diff
@@ -25,4 +25,18 @@ The installable mediums part of the releases are generated with the methods desc
 
 <Info>
 ** the `ubuntu` flavor tracks the latest available Ubuntu release (at the time of writing 22.10). the LTS flavors instead are tracking the latest LTS available in dockerhub. i.e. ubuntu-22-lts uses 22.04 as base image
-</Info>
\ No newline at end of file
+</Info>
+
+<Info>
```

Comment:
argh, my bad - we need to remove the <info> here, that was an oversight when switching from astro to docsy. The correct syntax is https://github.com/kairos-io/kairos/blob/master/docs/content/en/docs/Advanced/build.md?plain=1#L8

Reply:
https://www.docsy.dev/docs/adding-content/shortcodes/#alert